### PR TITLE
ICDS instance should have different text

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?243602#1258909

@benrudolph hope we like deploying to NIC!

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -870,6 +870,8 @@
 
 ONBOARDING_DOMAIN_TEST_DATE = ()
 
+HQ_INSTANCE = 'development'
```
Comment (benrudolph) [inline settings.py]: 👍 seems fine with me

**Review by benrudolph — APPROVED** [review]

File: corehq/apps/dashboard/templates/dashboard/dashboard_domain.html [hunk 2]
```diff
@@ -55,7 +57,14 @@
         {% endfor %}
     </div>
     <p class="lead text-center" style="margin-bottom: 5em;">
-        {% if project.commtrack_enabled %}
+        {% if is_icds %}
+            {% blocktrans %}
+                Thanks for using ICDS-CAS! If you have questions, suggestions,
+                or other thoughts, <br />we invite you to join the discussion on the
+                <a href="https://groups.google.com/forum/#!forum/commcare-users/join"
+                   target="_blank">CommCare Users Mailing List</a>.
```
Comment (czue) [inline corehq/apps/dashboard/templates/dashboard/dashboard_domain.html]: I suspect they might not want the content related to commcare users. @sravfeyn @mkangia do you know? seems easier to just omit.

**Comment by czue** [comment]
should squash merge this when ready in case it needs patching

File: corehq/apps/dashboard/templates/dashboard/dashboard_domain.html [hunk 3]
```diff
@@ -37,7 +37,9 @@
 {% block dashboard-content %}
 <div class="page-header">
     <h1>
-        {% if project.commtrack_enabled %}
+        {% if is_icds %}
+            {% trans 'Welcome Back to ICDS-CAS' %}
+        {% elif project.commtrack_enabled %}
```
Comment (czue) [inline corehq/apps/dashboard/templates/dashboard/dashboard_domain.html]: (feel free to delete this fork as well - we just discussed in #product)

Comment (emord) [inline corehq/apps/dashboard/templates/dashboard/dashboard_domain.html]: https://github.com/dimagi/commcare-hq/pull/14176/commits/beec929dc890cb93ece6b530fe4ebc8d659364ad :skull:

File: corehq/apps/dashboard/templates/dashboard/dashboard_domain.html (hunk 2 above)
Comment (emord) [inline corehq/apps/dashboard/templates/dashboard/dashboard_domain.html]: https://github.com/dimagi/commcare-hq/pull/14176/commits/beec929dc890cb93ece6b530fe4ebc8d659364ad

**Review by benrudolph — APPROVED** [review]

**Review by czue — APPROVED** [review]

**Comment by czue** [comment]
squash merge!

**Pull request merged**